File: DBtoCSharp_MARIADB/output/TABLE_STATISTICS.ts
export interface TABLE_STATISTICS
    {
        /**///  */
        TABLE_SCHEMA: string;
        /**///  */
        TABLE_NAME: string;
        /**///  */
        ROWS_READ: number;
        /**///  */
        ROWS_CHANGED: number;
        /**///  */
        ROWS_CHANGED_X_INDEXES: number;
}

File: DBtoCSharp_MARIADB/modelos/TABLE_STATISTICS.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Models
{
    public class TABLE_STATISTICS
    {
            [Required]
    [StringLength(192)]
        /// <summary>
        /// 
        /// </summary>
        public string TABLE_SCHEMA { get; set; }
            [Required]
    [StringLength(192)]
        /// <summary>
        /// 
        /// </summary>
        public string TABLE_NAME { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long ROWS_READ { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long ROWS_CHANGED { get; set; }
            [Required]
        /// <summary>
        /// 
        /// </summary>
        public long ROWS_CHANGED_X_INDEXES { get; set; }

}
}
